src/components/molecules/InboxItem.vue

```diff
@@ -5,7 +5,7 @@
     @keyup.enter="selectInboxItem"
     :class="[
       inboxItem.selected
-        ? 'bg-blue-selected rounded-none'
+        ? 'md:bg-blue-selected md:rounded-none'
```

Comment:
Desktop view now uses the sm (>= 640 px) breakpoint, not the md (>= 768 px) breakpoint.

Reply:
Thank you, I didn't realise this had changed